### PR TITLE
chore: Add finalization changes

### PR DESCRIPTION
**Pull request by efeDaniels**
- Updated PR message
- Updated Readme
- Updated PR Template
- Created a logos file and added some examples in it
- Changed logo link type to this repo for some assets
- Fixed a tiny bug about tags

### CODE REVIEW

**Comment by efeDaniels** [comment]
Thank you for your pull request! We will review it as soon as possible.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ _Before creating a pull request, please make sure read our [instructions](https:
...
 Project Name:
-Project Website:
+Twitter Handle:
 ```
 
 **Contact Information:**
 
 ```
 Point of Contact Telegram:
+Did you add @__ as a contact?:
```
Comment (obiedobo) [inline .github/pull_request_template.md:16]: Should this say @___ or something else?

File: README.md [hunk 2]
```diff
@@ -39,11 +39,13 @@ Submitting a PR is as simple as following these steps:
 
 3. **Edit the `.ts` file:** Locate and edit the applicable `.ts` file inside the `config` folder. Make sure to provide all the necessary information about your project.
 
-4. **Run configuration:** Run `yarn configure` to convert your TypeScript entry into a consumable JSON file.
+4. **Add your logo** Add your token's logo in .svg or .png format. The size of your logo should be 200 x 200 pixels, less than 50 KB and in the form of a circle.
```
Comment (obiedobo) [inline README.md:42]: Let's have this say something more like "should be square or circular and at least 200 x 200 pixels if PNG, ideally under 50KB"

File: README.md [hunk 3]
```diff
@@ -39,11 +39,13 @@ Submitting a PR is as simple as following these steps:
...
 
-5. **Commit and push:** Commit your changes and push them to your forked repository.
+5. **Run configuration:** Run `yarn configure` to convert your TypeScript entry into a consumable JSON file.
 
-6. **Submit a PR:** Create a new pull request in our `main` repository, targeting the main branch.
+6. **Commit and push:** Commit your changes and push them to your forked repository.
+
+7. **Submit a PR:** Create a new pull request in our `main` repository, targeting the main branch.
```
Comment (obiedobo) [inline README.md:48]: We should have them target the staging branch first actually!

File: README.md [hunk 4]
```diff
@@ -84,13 +94,29 @@ Once you finish filling, it should look like that:
       '0xabd7853b79e488bc1bd9e238a870167b074eb714',
       '0x71c0c1001520e1568e17836cc8a19d0dbdb2bd5f',
     ],
+    tags: ['DEX', 'Yield'],
```
Comment (obiedobo) [inline README.md:97]: Let's add Bonds or the closest tag to that here & in the actual listing

File: README.md [hunk 5]
```diff
@@ -84,13 +94,29 @@ Once you finish filling, it should look like that:
...
   },
 ```
 
+Lastly upload your logo and locate it into `logos` folder and make sure it's 200 x 200 pixels,has less than 50 KB size and in the form of a circle. Please name your logo's image with all uppercase letters and update the last part of `tokenLogoUrl`.
+
+```typescript
+    tokenLogoUrl: 'https://raw.githubusercontent.com/ApeSwapFinance/lhd-config/main/logos/YOURLOGONAME.svg',
+```
+
 **Be Aware Of Those Importnat Details:**
```
Comment (obiedobo) [inline README.md:114]: Misspelling, can you run the whole text through a grammar and text checker?

File: README.md [hunk 6]
```diff
@@ -84,13 +94,29 @@ Once you finish filling, it should look like that:
...
 
 `tokenSymbol` has to be always uppercase.
 
+Logo image's name has to be always uppercase, 200 x 200 px, and less than 50 KB.
```
Comment (obiedobo) [inline README.md:118]: You mention this 3 times, maybe find a way to consolidate

File: logos/KAI.svg
Comment (obiedobo) [inline logos/KAI.svg:1]: Do we use this logo?

File: config/whitelist.json [hunk 7]
```diff
@@ -981,7 +1034,7 @@
   {
     "tokenSymbol": "LGX",
     "tokenName": "LegionX",
-    "tokenLogoUrl": "https://raw.githubusercontent.com/ApeSwapFinance/apeswap-token-lists/main/assets/STZ.svg",
```
Comment (obiedobo) [inline config/whitelist.json:984]: Wrong logo

File: README.md (hunk 5 above)
Comment (obiedobo) [reply README.md:114]: Importnat

**Pull request merged**